src/Plugin/Hal.php

```diff
@@ -218,8 +283,6 @@ public function getHydratorManager()
     }
 
     /**
-     * Retrieve the metadata map
-     *
```

Comment:
I do not see the point of adding comments that does nothing more than the name of the method itself. I am in favor of minimalist methods with meaningful names that make unnecessary comments.